doc/recipes/flavor_registry/audio.md

```diff
@@ -0,0 +1,18 @@
+# Audio
+
+## Looping and Stopping Audio with AudioPlayers
+
+```dart
+import 'package:audioplayers/audioplayers.dart';
+
+AudioPlayer? loopPlayer;
+  
+Future<void>startSound()async{
+    loopPlayer = await FlameAudio.loopLongAudio('mylastsound.mp3');
+}
+
+void stopSound(){
+    loopPlayer!.stop();
+}
+```
+- Credit @viktorManual from [Discord Flame Channel](https://discord.com/channels/509714518008528896/516639688581316629/1012562750725169173)
```

Comment:
@spydon Do we want to include attribution links like this? I'm worried that it could create barriers against improving the recipes. Like, if the recipe is tagged as "belonging to user X", then people could hesitate to change it in any significant way.

Reply:
True true, it would quickly become hard to maintain.
Better to skip these ones and just ask people if code is copied from somewhere else into the recipes.

Reply:
Not certain if it would prevent people from updating it, but as they were snippets, I was wanting to provide context if there was any as to why it was written the way it was and what the person was solving.  Additionally, I was hoping it would encourage people to contribute to the community.

Reply:
It is better to explain the context directly next to the snippet, so that the reader wouldn't be forced to visit another site and dig through comments. Plus, I'm not even sure how stable the Discord comment links are.

> Additionally, I was hoping it would encourage people to contribute to the community.

Indeed, this does provide some degree of encouragement. But at the same time, it also serves as discouragement for anyone to modify/improve existing snippets.

Reply:
Ok, let me rework the blocks a bit.  I updated the section to be called Community Snippets but it can be just Snippets if we are pulling any mentions or links.  Additionally, this all started fit me because I felt the examples were lacking, but wow have they been rounded out.  I'm starting to question the need for snippets all together. The one thing that I really like about them though, is embedding the app in the docs. It makes it real nice and clean.  Having to the leave the docs to go to examples and then having to open another window for code is tedious, so maybe there is room for snippets. 

Up to you guys though.

Reply:
Maybe there is no need for a separate section for snippets? But we could have live examples everywhere in the docs where we are describing the features. Quite a big job though because I think most examples on the examples page are made for bigger screens.

Maybe there are cases where the snippets are more advanced than just involving one feature though, not sure?

Reply:
I really like the embeds.  The examples (amazing how many there are now) are too far away from the docs imo.  I consistently see you giving the examples link in discord and people are like, oh, I didn't know that was there.  So either every section in the docs needs a link to examples or there is an embed (huge task and may not be possible for every area) or there are snippets.

I do think there is a need for snippets (and my original intention) for common patterns that aren't exactly an explanation of Flame, but more for common patterns.  Like the how do I stop audio or how do I arc a bullet, etc.  It could even be how do I store my level data or such.

Reply:
>... or there is an embed (huge task and may not be possible for every area) or there are snippets.

I guess this will be the goal in the long run.

> I do think there is a need for snippets (and my original intention) for common patterns that aren't exactly an explanation of Flame, but more for common patterns.  Like the how do I stop audio or how do I arc a bullet, etc.  It could even be how do I store my level data or such.

Sounds good!

Reply:
I like the idea of having a small example embed in the docs for every Flame feature or component. This is similar to how Flutter has video examples for every widget -- only better, because we'll have a real live code.

I also agree with @munsterlander that recipes/snippets would be a separate form of documentation -- focused not on individual Flame features, but rather on how they fit together, or how they can be used to achieve specific game-development-related goals. These should ideally also feature embeds (where feasible).

The third kind of documentation is the Tutorials -- these would focus on complete games, or maybe just some large-ish pieces of functionality.

What's a bit unclear to me is how the current Examples would fit into the picture. Should it be a separate category, or is it better to split them across the 3 kinds of docs mentioned above?

Reply:
Well, I really like the examples, but I don't like how far apart they are from the docs.   Same with the api.  It would be awesome that when you are in the flame docs looking at say joystick component, you have the embed as well a links to the example and the api docs.

I think there is room for everything but they need to be connected so it's cohesive. Like you were saying about flutter docs.  It's one place for everything.

Reply:
Ok, now the docs redo and the tutorials and such have been sorted, I want to come back to this.  I really think it should be community snippets and like @st-pasha is saying, it should be:

 _focused not on individual Flame features, but rather on how they fit together, or how they can be used to achieve specific game-development-related goals. These should ideally also feature embeds (where feasible)._

So with that in mind, I am going to scrap a lot of what was in this PR and redirect it to this goal.  This would still leave tutorials and examples completely intact, but give a small quick area for short abstract concepts using Flame or other gaming concepts.